tests/core/gen_helpers/requirements.txt

```diff
@@ -1,2 +1,3 @@
 ruamel.yaml==0.16.5
 eth-utils==1.6.0
+pytest==3.6.1
```

Comment:
Why `3.6`?

Reply:
Ah nice catch! I was wrongly using the dependencies versions of the deposit contract tester.
Changed it to `>=4.4` as pyspec now.